platforms/extensibility/plugin-development/src/main/java/org/gradle/plugin/devel/plugins/JavaGradlePluginPlugin.java

```diff
@@ -172,7 +173,7 @@ private static void applyDependencies(Project project) {
             return;
         }
         DependencyHandler dependencies = project.getDependencies();
-        dependencies.add(API_CONFIGURATION, dependencies.gradleApi());
+        dependencies.add(JvmConstants.COMPILE_ONLY_API_CONFIGURATION_NAME, dependencies.gradleApi());
```

Comment:
@wakingrufus @eskatos 
Sorry I'm late to the party here. Any specific reason why you used `compileOnlyApi` instead of just `compileOnly`? 

I'm doing some verification on the api dependencies and was surprised to see `gradleApi()` showing up in the dependencies. See https://github.com/GradleUp/Tapmoc/issues/61